### PR TITLE
Improve "Hacking Wifi with Deauthentification Attack"

### PR DESCRIPTION
**Pull request by ctmbl**
Fix typos, errors and approximations

### CODE REVIEW

**Review by ctmbl — COMMENTED** [review]
@Clems1236 @ViSaN2k 
any comments on the title/summary improvements?

File: src/content/posts/wifi-deauth-attack.md/index.md [hunk 1]
```diff
@@ -13,35 +13,40 @@ draft: false
...
-To acheive this we must:  
-1) Detect WiFi
+To achieve this we must:  
+1) Detect target WiFi signal
 2) Detect connected computer to this WiFi
-3) Send a huge quantity of package to this computer to make it disconnect from the WiFi
-4) Get the password hash when it tries to reconnect to WiFi
-5) bruteforce it :)
+3) Send de-authentication packets to this computer to make it disconnect from the WiFi
+4) Get the password "hash" when it tries to reconnect to WiFi
```
Comment (Clems1236) [inline src/content/posts/wifi-deauth-attack.md/index.md:20]: 5) Get the MIC (Message Integrity Code) wich is a **hash** of several knownable informations included in the network capture and **password**
See the **4 Way Handshake** article for more informations

**Review by Clems1236 — CHANGES_REQUESTED** [review]

File: src/content/posts/wifi-deauth-attack.md/index.md [hunk 2]
```diff
@@ -89,42 +100,46 @@ Choose the one you want to hack and note the station MAC address.
...
 
-If deauthentification and reauthentification forced worked, we should have received EAPOL package. In listing package terminal window, there's EAPOL written in package received.
+If de-authentication and re-authentication forced worked, we should have received an EAPOL packet. In listing packet terminal window, there's EAPOL written in packet received.
 
 ![airodumpattack](airodumpattack.png)
 
-Deauth worked most of the time but immediate reauthentification doesn't always work. Indeed, Mac OS does not reconnect immediatly to previously reconnected Network after WiFi card unplanned restart. Windows OS doesn't follow the rule and try to reconnect instantly. (Prefer Windows Station to deauth if you have the choice ;))
+De-authentication works most of the time but immediate re-authentication doesn't always work.
+Indeed, Mac OS does not reconnect immediatly to previously reconnected Network after WiFi card unplanned restart.
+Windows OS doesn't follow the rule and try to reconnect instantly (prefer Windows station to attack if you have the choice :wink:)
```
[the remark this answers is not in the document]
Comment (Clems1236) [reply src/content/posts/wifi-deauth-attack.md/index.md:132]: No source available online, just test & try observations

**Review by Clems1236 — APPROVED** [review]
temporary approve

**Pull request merged**